flight_booker/src/Main.gren

```diff
@@ -194,11 +218,7 @@ parseDate dateString =
 
 formatDate : Date -> String
 formatDate date =
-    String.fromInt date.day
-        ++ "."
-        ++ String.fromInt date.month
-        ++ "."
-        ++ String.fromInt date.year
+    String.fromInt date.day ++ "." ++ String.fromInt date.month ++ "." ++ String.fromInt date.year
```

Comment:
I guess this one is tricky.

In this concrete case, I'd rather this being broken up in separate lines. But simple cases such as `(a ++ b)` looks better in a single line.

Reply:
Similar to single-element arrays and records, I could try preferring single-line if there's only one operator in the sequence.